src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -192,16 +190,17 @@ AtlasEngine::AtlasEngine()
     _sr.isWindows10OrGreater = IsWindows10OrGreater();
 
 #ifndef NDEBUG
-    // If you run the Host.EXE project inside Visual Studio it'll have a working directory of
-    // $(SolutionDir)\src\host\exe. This relative path will thus end up in this source directory.
-    _sr.sourceCodeWatcher = wil::make_folder_change_reader_nothrow(SHADER_SOURCE_DIRECTORY, false, wil::FolderChangeEvents::FileName | wil::FolderChangeEvents::LastWriteTime, [this](wil::FolderChangeEvent, PCWSTR path) {
-        if (til::ends_with(path, L".hlsl"))
-        {
-            auto expected = INT64_MAX;
-            const auto invalidationTime = std::chrono::steady_clock::now() + std::chrono::milliseconds(100);
-            _sr.sourceCodeInvalidationTime.compare_exchange_strong(expected, invalidationTime.time_since_epoch().count(), std::memory_order_relaxed);
-        }
-    });
+    {
+        _sr.sourceDirectory = std::filesystem::path{ __FILE__ }.parent_path();

```

Comment:
How the devil does this work when the app is packaged and deployed? Does it just look for the hlsl's in the package?